packages/app-page-builder/src/editor/plugins/elementSettings/action/ActionSettings.tsx

```diff
@@ -0,0 +1,228 @@
+                    const actionTypeOptions = [
+                        { id: "link", name: "Link" },
+                        { id: "onClickHandler", name: "Click Handler" }
+                    ];
+
+                    return (
+                        <>
+                            <Wrapper label={"Action Type"} containerClassName={classes.gridClass}>
+                                <Bind name="actionType" defaultValue={actionTypeOptions[0].id}>
+                                    <RadioGroup label="Select the button's action type">
+                                        {({ onChange, getValue }) => (
+                                            <React.Fragment>
+                                                {actionTypeOptions.map(({ id, name }) => (
+                                                    <Radio
+                                                        key={id}
+                                                        label={name}
+                                                        value={getValue(id)}
+                                                        onChange={onChange(id)}
+                                                    />
+                                                ))}
+                                            </React.Fragment>
+                                        )}
+                                    </RadioGroup>
+                                </Bind>
+                            </Wrapper>
+                            {actionType === "onClickHandler" ? (
+                                <>
+                                    <Wrapper
+                                        label={"Click Handler"}
+                                        containerClassName={classes.gridClass}
+                                    >
+                                        <Bind name={"clickHandler"}>
+                                            {({ value, onChange }) => {
+                                                const selectOptions = clickHandlers.map(plugin => {
+                                                    return {
+                                                        id: plugin.name,
+                                                        name: plugin.title
+                                                    };
+                                                });
+                                                return (
+                                                    <AutoComplete
+                                                        value={selectOptions.find(
+                                                            option => option.id === value
+                                                        )}
+                                                        placeholder="Select a handler"
+                                                        options={selectOptions}
+                                                        onChange={option => {
```

Comment:
Can't we just pass `onChange` here, instead of `option => { ... }`?

Reply:
We can. I'm more consistently debugging directly in chrome instead of peppering the code with console.logs now (which is why this was a function - I wanted to see what was passed to the onChange function so added a console.log here). #everydayisaschoolday :)